fluffy/views.py

```diff
@@ -264,11 +324,3 @@ def paste():
         })
     else:
         return redirect(paste_obj.url)
-
-
-@app.route('/upload-history')
-def upload_history():
-    return render_template(
-        'upload-history.html',
-        icon_extensions=ICON_EXTENSIONS,
-    )
```

Comment:
It would be nice if we can avoid moving these functions around since it's making this diff pretty hard to read (hard to tell what the actual changes are).